flink-runtime/src/main/java/org/apache/flink/runtime/io/network/buffer/NetworkBufferPool.java

```diff
@@ -69,14 +71,19 @@
 
 	private int numTotalRequiredBuffers;
 
+	private final int numberOfRequestedBuffers;
```

Comment:
numberOfRequestedBuffers -> numberOfRequiredBuffers?

Reply:
I would name it `numberOfSegmentsToRequest`